react-native-gutenberg-bridge/ios/GutenbergBridgeDelegate.swift

```diff
@@ -64,6 +64,10 @@ extension RCTLogLevel {
     }
 }
 
+public enum GutenbergUserEvent: String {
```

Comment:
I think that if the bridge is taking care of turning the raw string into a typed event, we might also benefit from adding stronger typing to the properties here. Otherwise we have one `guard` here that will bail if it doesn't recognize the name, and another in the WPiOS PR which bails if there is a `pageTemplateApplied` event but the `template` property is missing.

I'd probably go with an enum with associated values here and a custom initializer.